OpenRA.Game/Server/Server.cs

```diff
@@ -1008,12 +1168,43 @@ public void StartGame()
 			// TODO: Enable for multiplayer (non-dedicated servers only) once the lobby UI has been created
 			LobbyInfo.GlobalSettings.GameSavesEnabled = Type != ServerType.Dedicated && LobbyInfo.NonBotClients.Count() == 1;
 
+			// Player list for win/loss tracking
+			// HACK: NonCombatant and non-Playable players are set to null to simplify replay tracking
```

Comment:
It might be helpful for others if we add an extra line here `// the null padding is needed to keep the player indexes in sync with `world.Players` on the clients`?